app/views/teaching_events/show/_other-events.erb

```diff
@@ -0,0 +1,8 @@
+<%= render Content::InsetTextComponent.new(
+  title: "Find more events",
+  color: "grey",
+  text: <<~HTML
+    <p>We have lots more online and in-person events where you can get your questions answered about teaching.</p>
```

Comment:
@EllieNodder Very minor suggestion - given that this will appear on provider events as well as DfE events, do we need to change `We have` to be a bit more neutral eg `There are`? Otherwise it will seem like the `We` is the provider?